plugins/vivisect-broker/broker/vivserver.proto

```diff
@@ -8,12 +8,18 @@ message RunInstructionsRequest {
     int32 numInstructions = 1;
 }
 
+message MemoryAccess {
```

Comment:
nit: can you move this `message` definition into `RunInstructionReply` so it's the same as `message Instruction`?